build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -181,7 +182,12 @@ export class CheckPolicy extends BaseCommand<typeof CheckPolicy> {
 		const handlerExclusions: HandlerExclusions = {};
 		if (rawHandlerExclusions) {
 			for (const rule of Object.keys(rawHandlerExclusions)) {
-				handlerExclusions[rule] = rawHandlerExclusions[rule].map((e) => new RegExp(e, "i"));
+				const rawHandlerExclusionsRule = rawHandlerExclusions[rule];
+				assert(
+					rawHandlerExclusionsRule !== undefined,
+					"rawHandlerExclusionsRule is undefined in CheckPolicy.run()",
+				);
+				handlerExclusions[rule] = rawHandlerExclusionsRule.map((e) => new RegExp(e, "i"));
 			}
```

Comment:
Can you use `Object.entries` pattern here? We do not want to introduce a bunch of `assert()` into codebase.